azure-pipelines.yml

```diff
@@ -28,12 +28,11 @@ jobs:
 
   - script: |
       python -m pip install --upgrade pip
-      pip install --prefer-binary -r requirements-dev.txt
       pip install pytest-azurepipelines codecov
     displayName: 'Install dependencies'
 
   - script: |
-      pip install .
+      pip install --prefer-binary .[dev]
```

Comment:
Same here

Reply:
I'm not actually sure what `--prefer-binary` is for. The pip documentation says it's to "[p]refer older binary packages over newer source packages." Does that mean the goal here is to install the oldest versions of dependencies?